src/core/layout/scrollable-list/scrollable-list.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @module core/layout/scrollable-list
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0 Apache-2.0
+ */
+import { assetKey } from "./scrollable-list-helpers.js";
+import { createGelButton } from "./scrollable-list-buttons.js";
+
+export const scrollableList = scene => {
+    const panelConfig = lib.getPanelConfig(scene);
+    const scrollableListPanel = scene.rexUI.add.scrollablePanel(panelConfig).layout();
+    scene.input.topOnly = false;
+    return scrollableListPanel;
+};
+
+export const getPanelConfig = scene => {
+    const config = scene.config;
+    const { assetKeys: keys } = config;
+    const safeArea = scene.layout.getSafeArea();
+
+    return {
+        x: 0,
+        y: 0,
+        width: safeArea.width,
+        height: safeArea.height,
+        scrollMode: 0,
+        background: scene.add.image(0, 0, assetKey(keys.background, keys)),
+        panel: {
+            child: lib.createPanel(scene),
+            mask: {
```

Comment:
Totally minor but some of this can maybe be squashed up. Prettier will leave it alone if it fits on one line...